docs/man/docker-run.1.md

```diff
@@ -232,7 +232,8 @@ ENTRYPOINT.
    Read in a line delimited file of labels
 
 **--link**=[]
-   Add link to another container in the form of <name or id>:alias
+   Add link to another container in the form of <name or id>:alias or just <name or id>
+if the alias matches the name
```

Comment:
This needs to be updated in run.md too I think - https://github.com/docker/docker/blob/master/docs/sources/reference/run.md